app/views/service_manual/index.html.erb

```diff
@@ -18,15 +18,13 @@
           text: "Service Manual",
           font_size: "xl",
           heading_level: 1,
-          inverse: true,
           margin_bottom: 6,
         } %>
         <% lead_para = capture do %>
-          Helping teams to create and run great public services that meet the <%= link_to "Service Standard", "/service-manual/service-standard", class: "govuk-link govuk-link--inverse" %>.
+          Helping teams to create and run great public services that meet the <%= link_to "Service Standard", "/service-manual/service-standard", class: "govuk-link gem-c-inverse-header__link" %>.
```

Comment:
Martin and I already discussed this change - the new name of the class proposed here is potentially unclear and/or doesn't align with the naming convention used for the inverse link styles. Although we plan to rename the inverse header component at a later point at which point the new class name might more sense.

Will be good to get futher thoughts from Martin on this though 👀